services/QuillLMS/client/app/bundles/Teacher/components/college_board/scrollBox.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import scrollIntoView from 'smooth-scroll-into-view-if-needed'
+
+import { handleSetActiveSection } from '../../helpers/collegeBoard';
+import { ScrollSection } from '../../../../interfaces/collegeBoard';
+
+const ScrollBox = ({ activeSection, sections, setActiveSection, setIsScrollingFromClick }) => {
+
+  function handleClick(ref: React.RefObject<HTMLDivElement>, title: string) {
+    setIsScrollingFromClick(true);
+    handleSetActiveSection(title, setActiveSection);
+    scrollIntoView(ref.current, { behavior: 'smooth' }).then(() => {
+      setIsScrollingFromClick(false);
+    })
+  }
+
+  return(
+    <div className="scrollbox-container">
+      <section className="header">Scroll to</section>
+      <section className="sections-container">
+        {sections.map((section: ScrollSection) => {
+          const { ref, title, count }  = section;
+          const countStyle = !count ?  'hidden' : '';
+          const statusStyle = title !== activeSection ?  'inactive' : '';
```

Comment:
this is small, but I think it's better to start with the truthy statement - so `countStyle = count ? '' : 'hidden'` and same for `statusStyle`.

Reply:
fair point!